package.xml

```diff
@@ -10,4 +10,6 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <run_depend>vizbox</run_depend>
```

Comment:
Why is this the only dependency? Seems kind of random... Rest of the dependencies is in the bringup stuff?

Reply:
@LoyVanBeek

Reply:
I think so. I did not want to touch the other deps, only for what I add.

Reply:
Would be more consistent to move this dep to the targets.

Reply:
Did it myself. Not yet added in `tue-env-targets`